pytest_selenium/drivers/remote.py

```diff
@@ -30,9 +14,18 @@ def driver_kwargs(capabilities, firefox_profile, host, port, **kwargs):
     capabilities.setdefault('version', '')  # default to any version
     capabilities.setdefault('platform', 'ANY')  # default to any platform
 
-    executor = 'http://{host}:{port}/wd/hub'.format(host=host, port=port)
+    executor = 'http://{host}:{port}/wd/hub'.format(host=host or get_host(),
```

Comment:
Rather than adding `get_host` and `get_port` let's just do:

```python
host = host or os.environ.get('SELENIUM_HOST', 'localhost')
port = port or os.environ.get('SELENIUM_PORT', 4444)
```